### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Remove the line below if you want to inherit .editorconfig settings from higher directories
```
Comment (Lulalaby) [inline .editorconfig:1]: How did this file changed lol

File: DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs [hunk 2]
```diff
@@ -54,22 +54,22 @@ public class ApplicationCommandsConfiguration
...
-            this.ServiceProvider = provider;
+            this.ServiceProvider = Provider;
         }
 
         /// <summary>
         /// Creates a new instance of <see cref="ApplicationCommandsConfiguration"/>, copying the properties of another configuration.
         /// </summary>
-        /// <param name="acc">Configuration the properties of which are to be copied.</param>
-        public ApplicationCommandsConfiguration(ApplicationCommandsConfiguration acc)
+        /// <param name="Acc">Configuration the properties of which are to be copied.</param>
```
Comment (Lulalaby) [inline DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs:67]: ACC would be better or AppComConfig

File: DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs [hunk 3]
```diff
@@ -47,42 +47,42 @@ public sealed class ApplicationCommandsExtension : BaseExtension
         /// <summary>
         /// A list of methods for top level commands.
         /// </summary>
-        private static List<CommandMethod> _commandMethods { get; set; } = new List<CommandMethod>();
+        private static List<CommandMethod> CommandMethods { get; set; } = new List<CommandMethod>();
```
Comment (Lulalaby) [inline DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: privates should stick to _ I think, but we should discuss this.
I like it that way too

Comment (JBraunsmaJr) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#:~:text=When%20editing%20C%23%20code%20that%20follows%20these%20naming,t_.%20When%20writing%20method%20parameters%2C%20use%20camel%20casing.

File: DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs (hunk 2 above)
Comment (JBraunsmaJr) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs:67]: I do like acc

File: .editorconfig (hunk 1 above)
Comment (JBraunsmaJr) [reply .editorconfig:1]: aliens

File: DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs (hunk 3 above)
Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: oh

Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: can we apply this completly with .editorconfig and run code cleanup, like I did with the other things before?

Comment (JBraunsmaJr) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: honestly never fiddled with .editorconfig before sooooo mayyybeeee?

Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: Mhh let's look.

Comment (JBraunsmaJr) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: let's see how many hundreds of files will get updated this time lol

Comment (Lulalaby) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: we do it on this branch tho.
Don't need 120k changes on the other one xD

Comment (JBraunsmaJr) [reply DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs]: You don't want our browsers to break even more on the other one? lol

**Pull request closed without merging**